front-end/src/Pages/Glossary.elm

```diff
@@ -0,0 +1,1162 @@
+            Docs.metricInfo.other.metricDesc.compositionPRO
+            Docs.metricInfo.other.metricVarName.compositionPRO
+        , rowViewMetrics
+            Docs.metricInfo.other.metricName.compositionSER
+            Docs.metricInfo.other.metricDesc.compositionSER
+            Docs.metricInfo.other.metricVarName.compositionSER
+        , rowViewMetrics
+            Docs.metricInfo.other.metricName.compositionTHR
+            Docs.metricInfo.other.metricDesc.compositionTHR
+            Docs.metricInfo.other.metricVarName.compositionTHR
+        , rowViewMetrics
+            Docs.metricInfo.other.metricName.compositionTRP
+            Docs.metricInfo.other.metricDesc.compositionTRP
+            Docs.metricInfo.other.metricVarName.compositionTRP
+        , rowViewMetrics
+            Docs.metricInfo.other.metricName.compositionTYR
+            Docs.metricInfo.other.metricDesc.compositionTYR
+            Docs.metricInfo.other.metricVarName.compositionTYR
+        , rowViewMetrics
+            Docs.metricInfo.other.metricName.compositionVAL
+            Docs.metricInfo.other.metricDesc.compositionVAL
+            Docs.metricInfo.other.metricVarName.compositionVAL
+        , rowViewMetrics
+            Docs.metricInfo.other.metricName.compositionUNK
+            Docs.metricInfo.other.metricDesc.compositionUNK
+            Docs.metricInfo.other.metricVarName.compositionUNK
+        , rowViewMetrics
+            Docs.metricInfo.other.metricName.numberOfResidues
+            Docs.metricInfo.other.metricDesc.numberOfResidues
+            Docs.metricInfo.other.metricVarName.numberOfResidues
+        , rowViewMetrics
+            Docs.metricInfo.other.metricName.mass
+            Docs.metricInfo.other.metricDesc.mass
+            Docs.metricInfo.other.metricVarName.mass
+        ]
+
+
+view : Model -> Document Msg
+view model =
+    { title = "Glossary"
+    , body =
+        [ column
+            [ centerX
+            , spacing 20
+            , Style.pageWidths.singleColumn
+            , Font.size 16
+            ]
+            [ column [ spacing 20 ]
+                [ paragraph [] [ text """DE-STRESS uses a wide range of metrics that come from 
+                                         software developed by researchers all across
+                                         the world and are well used in protein design. However, due to the 
+                                         large numbers of metrics from different sources, it can be difficult 
+                                         to keep track of what these different metrics mean and 
+                                         how to use them. This page has been created to help you understand 
+                                         what these metrics mean, where they have came from and their convention 
+                                         for use.""" ]
+                , paragraph [] [ text """The first table gives a description of the different software
```

Comment:
"programmes" instead of software in this section might help the text to flow a bit better.

Reply:
I think I managed to find the DSSP version number just from the file name. EvoEF2 just has the version EvoEF V2 and I can't find anything for dfire2 (dfire-pair). I've checked the readme and looked in the code. I've also updated the text a bit as I used the word "different" quite a lot and changed software to programme.